src/main/scala/wolfendale/nunjucks/Frame.scala

```diff
@@ -57,9 +61,9 @@ final case class ChildFrame(values: Map[String, Value], parent: Frame) extends F
       resolve(key)
         .filterNot(_ == this)
         .map {
-          case _: RootFrame =>
-            // this case is strange
-            Frame(values + (key -> value), parent)
+//          case _: RootFrame =>
+//             this case is strange
+//            Frame(values + (key -> value), parent)
```

Comment:
Is this just temporary, or should this be removed?

Reply:
This should be removed, I'll do that now and remove the ignored test